context/.storybook/main.js

```diff
@@ -1,28 +1,31 @@
-const path = require("path");
-const {
-  alias
-} = require('../build-utils/alias');
+const path = require('path');
+const { alias } = require('../build-utils/alias');
 module.exports = {
-  "stories": ['../app/static/js/shared-styles/**/*.stories.js', '../app/static/js/components/**/*.stories.js'],
-  "addons": ["@storybook/addon-links", "@storybook/addon-essentials", 'storybook-addon-swc'],
-  "webpackFinal": async config => {
+  stories: [
+    '../app/static/js/shared-styles/**/*.stories.js',
+    '../app/static/js/shared-styles/**/*.stories.tsx',
+    '../app/static/js/components/**/*.stories.js',
+    '../app/static/js/components/**/*.stories.tsx',
```

Comment:
```suggestion
    '../app/static/js/**/*.stories.@(js|jsx|ts|tsx)'
```
Perhaps this could simplify it?